AsyncDisplayKit/ASMultiplexImageNode.mm

```diff
@@ -270,23 +270,25 @@ - (void)setDataSource:(id <ASMultiplexImageNodeDataSource>)dataSource
 
 - (NSArray *)imageIdentifiers
 {
-  OSSpinLockLock(&_imageIdentifiersLock);
-  NSArray *imageIdentifiers = [_imageIdentifiers copy];
-  OSSpinLockUnlock(&_imageIdentifiersLock);
-  return imageIdentifiers;
+  ASDN::MutexLocker l(_imageIdentifiersLock);
+  return _imageIdentifiers;
 }
 
 - (void)setImageIdentifiers:(NSArray *)imageIdentifiers
 {
-  OSSpinLockLock(&_imageIdentifiersLock);
+  _imageIdentifiersLock.lock();
```

Comment:
I think ideally we would use the brace-bounding "visual scope" style that is typically used with MutexLocker in this case; then the explicit unlock before the return can also disappear.  (I assume it is important to unlock before calling -fetchData)